### ISSUE TITLE
Landscape not working with ucal

### ISSUE THREAD

**Issue by amadeobellotti** [taalhavras/ucal#65]
Hi.

I went through the installation process and after I start %ucal-store and %calendar I see the calendar tab on Landscape. However when I click on it it brings up a page that says "Loading..." while my dojo is spamming out the following:

[%peek-path-is /x/~mopfel-winrux/calendars]
[%peek-path-is /x/~mopfel-winrux/calendars]
[%peek-path-is /x/~mopfel-winrux/calendars]

Does anyone have any idea how to fix this?

**Comment by finned-palmer** [comment]
I think the root cause here is the changes to graph-store that happened in Urbit in July. ucal needs to be updated to match the new scry paths.

**Comment by taalhavras** [comment]
Yeah, I've been meaning to fix this but I haven't had time to do it. If
anyone wants to take a shot feel free - I don't think this is supposed to
be too complex of an upgrade. Otherwise I'll get to it when I get some
spare cycles.
